Drew bakerfdm podcast episode 7 (#8567)
fleetdm/fleet: articles/the-future-of-device-management-ep7.md

# Future of device management
## EP 7: Mat X: Organizing community insights for industry growth with MacAdmin tools

<iframe title="Mat X, Organizing community insights for industry growth with MacAdmin tools" allowtransparency="true" height="300" width="100%" style="border: none; min-width: min(100%, 430px);" scrolling="no" data-name="pb-iframe-player" src="https://www.podbean.com/player-v2/?from=embed&i=7dyjk-13049d4-pb&square=1&share=1&download=1&fonts=Arial&skin=f6f6f6&font-color=auto&rtl=0&logo_link=episode_page&btn-skin=666666&size=300" allowfullscreen=""></iframe>

Listen to the episode on [Apple](https://podcasts.apple.com/us/podcast/mat-x-organizing-community-insights-for-industry-growth/id1641183838?i=1000584995357), [Spotify](https://open.spotify.com/episode/5wXMNulhPsXJXFpdPkk7YD?si=_6sSsyAfTAG9BamZKspwOA), or [PodBean](https://www.podbean.com/ew/pb-7dyjk-13049d4).

### Show notes: 

> In his day job Mat X serves the local film, VFX (visual effects), and media production industries. As an established storage, workflow and backup expert, he builds better workflows from camera to archive and back again. Working with high speed fiber channel SAN storage led to a healthy respect for well established, backup, archive and disaster recovery systems, balancing on premise LTO (linear tape open) systems which offer fast and secure backups with cloud (other people’s data centers).

Reviewer: ```suggestion
In his day job, Mat X serves the local film, visual effects (VFX), and media production industries. As an established storage, workflow, and backup expert, he builds better workflows from camera to archive and back again.

Working with high speed fiber channel SAN storage led to a healthy respect for well established backup, archive, and disaster recovery systems — balancing on-premises linear tape open (LTO) systems that offer fast and secure cloud backups.
```

> Many years of managing Macs and other devices has created a large community of IT professionals wanting to do the same. Seeing the new trend of Mac open source software collaboratively written and wanting to focus on the niche of developers (Dev) and IT (Ops) working together, Mat X founded a new conference called MacDevOps:YVR (MDO:YVR for short). In 2015, the first MDO annual conference of IT, system administrators, programmers and DevOps professionals worldwide took place in Vancouver (YVR). Knowing that more storage and services were moving into the “Cloud”, and that writing software collaboratively was a necessary learning goal for many, MDO was created as a way forward to building a better community. Several successful in person conferences later, the pandemic shifted everything online to a greater worldwide audience with speakers and attendees around the globe. Now with the MDO podcast, bookclub series, and many workshops online throughout the pandemic, the learning never stops. Planning continues for more conferences and community events to train the next generation of IT Pros and seasoned conference speakers.

Reviewer: ```suggestion
Many years of managing Macs and other devices has created a large community of IT professionals wanting to do the same. Mat X saw the new trend of Mac open-source software written through collaboration. He wanted to focus on the niche of developers (Dev) and IT (Ops) working together, so he founded a new conference called MacDevOps:YVR (MDO:YVR for short).

In 2015, IT, system administration, programming, and DevOps professionals from all over the world gathered in Vancouver (YVR) for the first MDO annual conference. Knowing that more storage and services were moving to the cloud — and that writing software collaboratively was a necessary learning goal for many — Mat X created MDO as a way forward to build a better community.
```

### Topics discussed:

- Mat’s background and how he got into the MacAdmins world
- What a day in the life of Mat X looks like
- Illumating the biggest changes of MacAdmin over the years
- The creation and journey of MDOYVR

> - Why MacAdmins came to be so community-oriented (open source eg. MicroMDM, Nudge, etc)

Reviewer: I'm a little confused by the parenthetical, "(open source eg. MicroMDM, Nudge, etc)." Does this mean that MicroMDM and Nudge are examples of open source?

Author: Changed it to "- Why MacAdmins came to be so community-oriented (Offering open source tools like MicroMDM, Nudge, etc)" I think that works better

Many years of managing Macs and other devices has created a large community of IT professionals wanting to do the same. Mat X saw the new trend of Mac open-source software written through collaboration. He wanted to focus on the niche of developers (Dev) and IT (Ops) working together, so he founded a new conference called MacDevOps:YVR (MDO:YVR for short).

In 2015, IT, system administration, programming, and DevOps professionals from all over the world gathered in Vancouver (YVR) for the first MDO annual conference. Knowing that more storage and services were moving to the cloud — and that writing software collaboratively was a necessary learning goal for many — Mat X created MDO as a way forward to build a better community.

### Topics discussed:

- Mat’s background and how he got into the MacAdmins world
- What a day in the life of Mat X looks like
- Illumating the biggest changes of MacAdmin over the years
- The creation and journey of MDOYVR

> - Why MacAdmins came to be so community-oriented (Offering open source tools like MicroMDM, Nudge, etc)

Reviewer: ```suggestion
- Why MacAdmins became so community-oriented, offering open-source tools like MicroMDM, Nudge, etc.
```
